homework-03/src/isPrime.js

```diff
@@ -0,0 +1,9 @@
+isPrime = (n) => {
+  const max = Math.sqrt(n);
+  for (let i = 2; i <= max; i++) {
```

Comment:
this loop is blocking on high values, it's already fine, but you might want to find a way to get rid of it? :)